src/tests/ftest/util/test_utils_container.py

```diff
@@ -372,7 +372,7 @@ def __init__(self, pool, daos_command, label_generator=None, namespace=CONT_NAME
         self.label = BasicParameter(None, "TestContainer")
         self.label_generator = label_generator
 
-        self.register_cleanup = BasicParameter(True)  # call register_cleanup by default
+        self.register_cleanup = BasicParameter(True, True)  # call register_cleanup by default
```

Comment:
what is the first argument here? I see other BasicParameter() invocations pass None as first, almost always. Is it not 'self' ?

Reply:
First argument is "initial" value and the second is "default value". I don't know the details of why it was implemented this way, but when the test gets parameters from the config, it overrides the "initial" with the "default". Basically rendering the "initial" useless in some cases.